srcpkgs/yacy/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'yacy'
+pkgname=yacy
+version=1.922
+revision=1
+wrksrc=${pkgname}
+hostmakedepends="openjdk apache-ant"
+depends="openjdk-jre"
+short_desc="YaCy p2p decentralized search engine"
+maintainer="Jason Manley <jason@jasondavid.tv>"
+license="GPL-3.0-or-later"
+homepage="https://yacy.net"
+distfiles="http://latest.yacy.net/yacy_v${version}_20191013_9964.tar.gz"
+checksum=@99af963642dafaeab2657ad59315e7b7896bf4c1e7091f8f67e95e71802d44df
+
+system_accounts="yacy"
+yacy_homedir="/var/lib/yacy"
+yacy_shell="/bin/bash"
+make_dirs="/var/lib/yacy 0755 yacy yacy"
```

Comment:
I suppose `/var/lib/yacy/startYACY.sh` will be installed into `/var/lib/yacy`.
Then, this is unnecessary.

Anyway, can that script installed somewhere else?
I don't think `/var` is suitable to install files?

`yacy_shell` is `/bin/bash`, then this package should depend on `bash`, no?